meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk.bb

```diff
@@ -4,6 +4,7 @@ require docker-disk.inc
 SUPERVISOR_REPOSITORY_armv5 = "resin/armel-supervisor"
 SUPERVISOR_REPOSITORY_armv6 = "resin/rpi-supervisor"
 SUPERVISOR_REPOSITORY_armv7a = "resin/armv7hf-supervisor"
+SUPERVISOR_REPOSITORY_armv7va = "resin/armv7hf-supervisor"
```

Comment:
I remember you saying the arch was **armv7ve**

Was that a typo and you meant instead **armv7va** ?